hue/hue/base/hue-ini-secret.yaml

```diff
@@ -0,0 +1,179 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hue-ini
+type: Opaque
+stringData:
+  hue.ini: |
+    [desktop]
+      # Hide unused apps
+      app_blacklist=impala,security,jobbrowser,jobsub,pig,hbase,sqoop,zookeeper,spark,oozie,search
+      secret_key=thisISaSECRET_1234
```

Comment:
What is this `secret_key`?

Reply:
It's a django app secret key

See: https://docs.cloudera.com/documentation/enterprise/5-7-x/topics/cdh_ig_hue_config.html#topic_15_6_2__title_371

Similar to the superser's secret key https://github.com/opendatahub-io/odh-manifests/blob/58b94891d0808fd333271a0a282b018af4e2ab81/superset/base/configmap.yaml#L24

I think we should rather have it loaded from the `params.env`, correct?

Reply:
Templated via `params.env`